versioned/persist/store/src/main/java/org/projectnessie/versioned/persist/store/PersistVersionStore.java

```diff
@@ -473,19 +476,21 @@ private Hash refToHash(Ref ref) throws ReferenceNotFoundException {
   }
 
   @Override
+  @SuppressWarnings("MustBeClosedChecker")
```

Comment:
instead of suppressing we should further delegate the `@MustBeClosed` on this method, no?

if there is a good reason for why we dont, then we should add this as a comment after the suppression